Builds/CMake/CMakeFuncs.cmake

```diff
@@ -147,11 +147,7 @@ macro(setup_build_cache)
     set(CMAKE_CONFIGURATION_TYPES
       DebugClassic
       ReleaseClassic)
-    if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
-      set(CMAKE_BUILD_TYPE DebugClassic)
-    elseif(${CMAKE_BUILD_TYPE} STREQUAL "Release")
-      set(CMAKE_BUILD_TYPE ReleaseClassic)
-    endif()
+    set(CMAKE_BUILD_TYPE "${CMAKE_BUILD_TYPE}Classic")
```

Comment:
Unity build is still broken (`-Dtarget=gcc.release.unity` does not define `NDEBUG` when it should). Probably a bug in the target parser. Looking...